instance/management/commands/kill_zombies.py

```diff
@@ -71,16 +71,18 @@ def handle(self, *args, **options):
 
         if not nova_servers:
             self.log('No servers found in region {}.'.format(self.region))
-            return
+            return None
         self.log('Found {} unterminated servers in region {}.'.format(len(nova_servers), self.region))
 
         # Scan each server for the zombieness.
         death_count = sum(1 for srv in nova_servers if self.not_zombie_or_die(srv))
 
         if self.dry_run:
-            self.log("Would have terminated {} zombies if this weren't a dry run.".format(death_count))
+            result = "Would have terminated {} zombies if this weren't a dry run.".format(death_count)
         else:
-            self.log("Terminated {} zombies.".format(death_count))
+            result = "Terminated {} zombies.".format(death_count)
+        self.log(result)
+        return result
```

Comment:
@spokerman12, these changes appear to just be a cleanup of some of the existing code. Are they needed for any of the other changes made in this PR? If not, I would recommend dropping these from the PR.

Reply:
These are needed as `result` is `call_command`'s return value.